### ISSUE TITLE
[Beta 1] Disabling all visible plugins causes MO to crash.

### ISSUE THREAD

**Issue by NexusAU** [TanninOne/modorganizer#685]
If I click 'Disable all visible' in plugins (seen here: http://i.imgur.com/GMsZDVT.png) it causes MO to crash. I can disable mods individually just fine however. Not a major issue as I can just do them individually but I thought I'd report it anyway.

Loving the update btw.

**Comment by Wadu436** [comment]
I have the same problem. When I reopen MO, there are always a lot of mods disabled but not all. I tried moving some mods and was able to see a pattern. The last mod disabled is always one with an esp or an esm. I attached the dump from one of the crashes to my post.

OS: Windows 10 Home 64-Bit
CPU: 6600k OC'ed to 4.4Ghz
GPU: GTX 970 OC'ed to 1393Mhz Boost and 1253Mhz Memory Boost
Ram: 8GB DDR4 2133Mhz

[ModOrganizerDmp.zip](https://github.com/TanninOne/modorganizer/files/257870/ModOrganizerDmp.zip)
